File: src/ActividadDinamica.js
function encontrarMaximo(a, b, c) {
    if (a >= b && a >= c) {
        return a;
    } else if (b >= a && b >= c) {
        return b;
    } else {
        return c;
    }
}

console.log("=== ¿CUÁL ES EL NÚMERO MÁS GRANDE? ===");

const num1 = parseInt(prompt("Ingresa el primer número: "));
const num2 = parseInt(prompt("Ingresa el segundo número: "));
const num3 = parseInt(prompt("Ingresa el tercer número: "));

const maximo = encontrarMaximo(num1, num2, num3);
console.log("-----------------------"); 
console.log("Resultado");
console.log("El número mayor es: " + maximo);

File: bin/ActividadDinamica.py
def encontrar_maximo(a, b, c):
    if a >= b and a >= c:
        return a
    elif b >= a and b >= c:
        return b
    else:
        return c

print("=== ¿CUÁL ES EL NÚMERO MÁS GRANDE? ===")

num1 = int(input("Ingresa el primer número: "))
num2 = int(input("Ingresa el segundo número: "))
num3 = int(input("Ingresa el tercer número: "))

maximo = encontrar_maximo(num1, num2, num3)
print("-----------------------") 
print("Resultado")
print(f"El número mayor es: {maximo}")
